### DISCUSSION TITLE
Why is the number of lists limited?

### DISCUSSION THREAD

**Discussion by ThallyssonKlein** [community/community#10233] [Other Feature Feedback, Questions, & Ideas] [answered]
I found the lists feature very useful and I'm using it a lot, but unfortunately I've reached the maximum number of lists and I can't create the lists I need anymore '-'

**Comment by bailey** [comment, marked as answer]
Hi @ThallyssonKlein! Thanks for your feedback. ✨  We have another discussion tracking work to raise the list limit here: https://github.com/github/feedback/discussions/8633 We will post updates there!

**Comment by rszymula** [comment]
Can we PLEASE get unlimited?

Lists were a AMAZING feature added, and we cannot get enough!

Please at 32 its still too short, unlimited would be great, but 128 should be enough!
So much easier to group repos that I want to reference later with LISTS!

**Reply by rszymula** [reply]
I am talking about the # of Lists you can have in the Starred Repos

**Comment by rszymula** [comment]
Ill pay extra $ to get more space in lists. If you really need justification for it, make it unlimited or 256 max for PAYING MEMBERS ONLY

**Reply by rszymula** [reply]
I am talking about the # of Lists you can have in the Starred Repos
